src/code/z_player_lib.c

```diff
@@ -315,27 +316,27 @@ void func_8012300C(PlayState* play, s32 arg1) {
     player->unk_B2B = arg1;
 }
 
-void func_8012301C(Player* player, PlayState* play2) {
+void func_8012301C(Actor* thisx, PlayState* play2) {
     PlayState* play = play2;
-    s32 pad;
+    Player* this = (Player*)thisx;
 
-    player->unk_AE7++;
+    this->unk_AE7++;
 
-    if (player->unk_AE7 == 2) {
-        s16 objectId = gPlayerFormObjectIndices[((void)0, gSaveContext.save.playerForm)];
+    if (this->unk_AE7 == 2) {
+        s16 objectId = gPlayerFormObjectIndices[GET_PLAYER_FORM];
```

Comment:
Does this `GET_PLAYER_FORM` work for all uses of `gSaveContext.save.playerForm` i.e. did you run into any regalloc issues?

Reply:
There are a few places where it messes the regalloc a bit :s